website/templates/project/project.mako

```diff
@@ -51,8 +51,10 @@
                             <a class="btn btn-default" href="#nodesPrivacy" data-toggle="modal">Make Private</a>
                         % endif
                         <button class="btn btn-default disabled">Public</button>
-                    % endif
                     </div>
+
+                    % endif
```

Comment:
It looks like the `% if` this is closing is contained within a `<div class="btn-group">`, so the closing `</div>` should be below this.